app/routes/circulars/index.tsx

```diff
@@ -204,14 +223,10 @@ export default function () {
         <Link to="/docs/circulars">documentation</Link> for help with
         subscribing to or submitting Circulars.
       </p>
-      <ButtonGroup className="position-sticky top-0 bg-white margin-bottom-1 padding-top-1">
-        <Form
-          className="display-inline-block usa-search usa-search--small"
-          role="search"
-        >
-          <Label srOnly={true} htmlFor="query">
-            Search
-          </Label>
+      <div className="position-sticky top-0 bg-white margin-bottom-1 padding-top-1">
+        <Form className="usa-search usa-search--small width-full">
+          <Label htmlFor="query">Search</Label>
```

Comment:
This label used to be for screen readers only (`srOnly={true}`). Why did you make it visible?

Reply:
this was removed by mistake and overlooked. Fixed in [45cf283](https://github.com/nasa-gcn/gcn.nasa.gov/pull/1104/commits/45cf2833f6c53d26fc129ec4e1942986c630024a)